ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/OpenStackInfrastructure.py

```diff
@@ -239,6 +232,9 @@ def collect(self, device, unused):
         except Exception, e:
             log.warning("Unable to determine nova URL for nova-api component discovery: %s" % e)
 
+        # not being able to resolve hostname could, in some cases,
+        # result in nova-api and/or cinder-api components not being modeled
+        # make sure /etc/hosts file on the controller host is decent
```

Comment:
Can you explain what you mean by this?  I wouldn't think the hosts file on the openstack controller would be important.